pkg/deployments/scaler.go

```diff
@@ -79,7 +79,7 @@ func (s *scaler) compareScales(current, desired int32) {
 		s.desiredScale = desired
 	}
 
-	if s.currentScale == -1 || s.desiredScale == -1 {
+	if s.currentScale == -1 || s.desiredScale == -1 || desired == -1 {
```

Comment:
good catch 👍

Reply:
It doesn't actually solve issue #67 . So I might  revert this